TestingLowerBounds/FDiv/CondFDiv.lean

```diff
@@ -471,6 +472,13 @@ lemma condFDiv_zero_measure : condFDiv f κ η 0 = 0 := by
   rw [condFDiv_eq' hf_ae integrable_zero_measure]
   simp only [integral_zero_measure, EReal.coe_zero]
 
+@[simp]
+lemma condFDiv_isEmpty_left [IsEmpty α] : condFDiv f κ η μ = 0 := by
+  have h : μ = 0 := by
+    ext s
+    exact Set.eq_empty_of_isEmpty s ▸ measure_empty
+  exact h ▸ condFDiv_zero_measure
```

Comment:
It does not make much of a difference, but the proof is perhaps slightly better that way:
```suggestion
  suffices μ = 0 from this ▸ condFDiv_zero_measure
  ext s
  exact Set.eq_empty_of_isEmpty s ▸ measure_empty
```